src/code/z_map_disp.c

```diff
@@ -1,109 +1,1813 @@
+}
+
+void func_80104F34(PlayState* play) {
+    s32 i1;
+    s32 i2;
+    s32 i3;
+    s32 i4;
+
+    for (i1 = 0; i1 < 32; i1++) {
+        D_801BEBB8.unk48[i1] = -0x7FFF;
+    }
+    for (i2 = 0; i2 < D_801BEC1C; i2++) {
+        MinimapEntry* mapEntry1 = &D_801BEBB8.minimapList->entry[i2];
+
+        if (mapEntry1->mapId == 0xFFFF) {
+            continue;
+        }
+        for (i1 = 0; i1 < 32; i1++) {
+            if (D_801BEBB8.unk48[i1] == -0x7FFF) {
+                D_801BEBB8.unk48[i1] = mapEntry1->unk4;
+                break;
+            } else if (fabsf((f32)D_801BEBB8.unk48[i1] - (f32)mapEntry1->unk4) < 5.0f) {
+                break;
+            }
+        }
+    }
+    for (i2 = 0; i2 < D_801BEC1C; i2++) {
+        if (D_801BEBB8.unk48[i2] == -0x7FFF) {
+            break;
+        }
+        for (i3 = i2 + 1; i3 < D_801BEC1C; i3++) {
+            if (D_801BEBB8.unk48[i3] == -0x7FFF) {
+                break;
+            }
+            if (D_801BEBB8.unk48[i3] < D_801BEBB8.unk48[i2]) {
+                s16 temp_t3 = D_801BEBB8.unk48[i2];
+
+                D_801BEBB8.unk48[i2] = D_801BEBB8.unk48[i3];
+                D_801BEBB8.unk48[i3] = temp_t3;
+            }
+        }
+    }
+    for (i2 = 0; i2 < D_801BEC1C; i2++) {
+        MinimapEntry* mapEntry2 = &D_801BEBB8.minimapList->entry[i2];
+
+        D_801BEBB8.unk3C[i2] = -1;
+
+        for (i4 = 0; i4 < D_801BEC1C; i4++) {
+            if (D_801BEBB8.unk48[i4] != -0x7FFF) {
+                if (fabsf((f32)D_801BEBB8.unk48[i4] - (f32)mapEntry2->unk4) < 5.0f) {
+                    D_801BEBB8.unk3C[i2] = i4;
+                    break;
+                }
+            }
+        }
+    }
+    D_801BEBB8.unk40 = 0;
+    for (i2 = 0; i2 < D_801BEC1C; i2++) {
+        if (D_801BEBB8.unk48[i2] != -0x7FFF) {
+            D_801BEBB8.unk40++;
+        }
+    }
+    D_801BEBB8.unk44 = 0;
+    for (i2 = 0; i2 < 5; i2++) {
+        if (play->sceneId == D_801BEC5C[i2].unk0) {
+            D_801BEBB8.unk44 = D_801BEC5C[i2].unk2;
+        }
+    }
+}
+
+s32 func_80105294(void) {
+    T_801BEC70 sp4 = D_801BEC70;
+
+    if ((D_801BEBB8.minimapList == NULL) || (D_801BEBB8.unk58 < 0) || (D_801BEBB8.unk58 >= 5) || (D_801BEC1C == 0)) {
+        return 0x7B;
+    }
+    return sp4.unk0[D_801BEBB8.unk58];
+}
+
+s16 func_80105318(void) {
+    return D_801BEBB8.unk58;
+}
+
+// TransitionActor params test
+s32 func_80105328(s32 params) {
+    if (ENDOOR_PARAMS_GET_TYPE((u16)params) == ENDOOR_TYPE_5) {
+        return true;
+    }
+    return false;
+}
+
+void func_8010534C(PlayState* play) {
+    TransitionActorList* transitionActors = &D_801BEC24;
+    s32 var_v0;
+    s32 i;
+
+    for (i = 0; i < transitionActors->count; i++) {
+        if (func_80105328(D_801F53B0[i].params) != 0) {
+            if (ABS_ALT(D_801F53B0[i].id) != 0x18) {
+                for (var_v0 = 0; var_v0 < D_801BEBB8.unk40; var_v0++) {
+                    s32 temp = (D_801BEBB8.unk48[var_v0] - 5); // needed for matching
```

Comment:
```suggestion
                    s32 temp = (D_801BEBB8.unk48[var_v0] - 5);
```
I don't think this is necessary? If you think it's fake, could put a fake label